lib/matplotlib/_api/__init__.py

```diff
@@ -305,7 +305,6 @@ def my_func(*args, **kwargs):
                 [lambda old1, old2: locals(), lambda new: locals()],
                 *args, **kwargs)
             if "old1" in params:
-                warn_deprecated(...)
```

Comment:
Why was this removed? This is example code for certain deprecation procedures, while the `...` is in place of meaningful contextual inputs to `mpl._api.warn_deprecated` the fact that you want to warn in some cases is expected.

This is also unrelated to `inset_axes`